docs/features/refout.md

```diff
@@ -67,10 +68,10 @@ As mentioned above, there may be further refinements after C# 7.1:
 - produce ref assemblies even when there are errors outside method bodies (emitting error types when `EmitOptions.TolerateErrors` is set)
 
 ## Open questions
-- should explicit method implementations be included in ref assemblies?
+- should explicit method implementations be included in ref assemblies? (answer: no. The interfaces that are declared as implemented are what matter to consuming compilations)
```

Comment:
Reverted the note. Keeping open question on explicit method implementations.

---
In reply to: [114207367](https://github.com/dotnet/roslyn/pull/19133#discussion_r114207367) [](ancestors = 114207367)